src/test/java/com/google/api/generator/engine/ast/MethodDefinitionTest.java

```diff
@@ -661,4 +662,14 @@ private static AssignmentExpr createAssignmentExpr() {
 
     return AssignmentExpr.builder().setVariableExpr(variableExpr).setValueExpr(valueExpr).build();
   }
+
+  private static List<CommentStatement> createCommentStatements() {
+    JavaDocComment javaDocComment =
+        JavaDocComment.builder()
+            .addComment("Constructs an instance of GrpcMyProtoStub, using the given settings.")
+            .addComment(
+                "This is protected so that it is easy to make a subclass, but otherwise, the static factory methods should be preferred.")
+            .build();
+    return new ArrayList<>(Arrays.asList(CommentStatement.withComment(javaDocComment)));
```

Comment:
Why do we need a new `ArrayList` wrapping a List?

Reply:
nice catch, `Arrays.asList()` only should work, thanks.